catalog-info.yaml

```diff
@@ -0,0 +1,24 @@
+# Catalog entry for Backstage [backstage.io]
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: eco-goinfra
+  title: eco-goinfra
+  description: Ecosystem QE framework
+  links:
+    - title : Ginkgo
```

Comment:
This set of tools doesn't use ginkgo

Reply:
Correct! Updated it to go.dev, so that it is in sync with Readme